### PR TITLE
Update the Artifacts for Openshift

### PR DESCRIPTION
**Pull request by asifdxtreme**
**What type of PR is this?**
 /kind enhancement

**What this PR does / why we need it**:
This PR standardises the artifacts required for Red Hat Certification

 /kind TESTED

### CODE REVIEW

File: artifacts/openshift/vsphere-kubernetes-drivers-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -106,11 +107,8 @@ spec:
           - description: 'Informs the status of vSphere Storage Provider'
             displayName: 'Storage Provider'
             path: csi
-  description: 'Vsphere Kubernetes Drivers Operator helps to install/manage vsphere drivers required to run kubernetes workload on vSphere. The operator currently manages CPI and CSI vSphere drivers. The operator exposes two resources: VsphereCloudConfig - represents the connection information required to connect to vcenter. VDOConfig - represents the configuration information w.r.t CPI and CSI'
-  displayName: Vsphere Kubernetes Drivers Operator
-  icon:
```
Comment (jvrahav) [inline artifacts/openshift/vsphere-kubernetes-drivers-operator.clusterserviceversion.yaml:111]: do we not require this anymore

File: artifacts/openshift/vsphere-kubernetes-drivers-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -106,11 +107,8 @@ spec:
...
-      mediatype: image/svg+xml
+  description: 'Vsphere Kubernetes Drivers Operator helps to install/manage vsphere drivers required to run kubernetes workload on vSphere.'
```
Comment (jvrahav) [inline artifacts/openshift/vsphere-kubernetes-drivers-operator.clusterserviceversion.yaml:110]: can we add the rest of the description as well.

File: artifacts/openshift/vsphere-kubernetes-drivers-operator.clusterserviceversion.yaml [hunk 3]
```diff
@@ -36,12 +37,12 @@ metadata:
     operators.operatorframework.io/builder: operator-sdk-v1.10.1
     operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
     categories: Integration & Delivery
-    containerImage: ""
+    containerImage: projects.registry.vmware.com/vsphere_kubernetes_driver_operator/vdo@sha256:4fb2c8add979f8d06a0aead4286d70f7edf33ec312638a30a68d4fc896e76463
```
Comment (jvrahav) [inline artifacts/openshift/vsphere-kubernetes-drivers-operator.clusterserviceversion.yaml:40]: will this sha change to the latest image that gets pushed to harbor repo

File: artifacts/openshift/vdo-controller-manager-metrics-service-rbac.yaml [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
```
Comment (jvrahav) [inline artifacts/openshift/vdo-controller-manager-metrics-service-rbac.yaml:1]: need to update Makefile as well for this change so the files are placed accordingly

**Review by jvrahav — CHANGES_REQUESTED** [review]

File: artifacts/openshift/vsphere-kubernetes-drivers-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (asifdxtreme) [reply artifacts/openshift/vsphere-kubernetes-drivers-operator.clusterserviceversion.yaml:111]: Icon length is too big, and the yaml  lint is failing this part, we need to find a way to put this in a different way, IMO as of now we can skip this

File: artifacts/openshift/vsphere-kubernetes-drivers-operator.clusterserviceversion.yaml (hunk 3 above)
Comment (asifdxtreme) [reply artifacts/openshift/vsphere-kubernetes-drivers-operator.clusterserviceversion.yaml:40]: No, the reason RH Certification tries to use SHA is that tag's can be deleted or created again, but SHA will be different if the tags has deleted and created again.

so to maintain the consistency, they use SHA

File: artifacts/openshift/vdo-controller-manager-metrics-service-rbac.yaml (hunk 4 above)
Comment (asifdxtreme) [reply artifacts/openshift/vdo-controller-manager-metrics-service-rbac.yaml:1]: yes, will do these changes

File: artifacts/openshift/vsphere-kubernetes-drivers-operator.clusterserviceversion.yaml (hunk 2 above)
Comment (asifdxtreme) [reply artifacts/openshift/vsphere-kubernetes-drivers-operator.clusterserviceversion.yaml:110]: the problem is with lint not taking anything above 180 characters in 1 line, will check if we can add multiple lines

**Pull request closed without merging**